core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java

```diff
@@ -396,8 +408,11 @@ public double getMaxspeed(EdgeIteratorState edge, int accessType, boolean revers
 
     @Override
     public long flagsDefault(boolean forward, boolean backward) {
-        // just pick car mode to set access values?
-        throw new RuntimeException("do not call flagsDefault");
+//         just pick car mode to set access values?
+//        throw new RuntimeException("do not call flagsDefault");
+    	// TODO This is called on each of the encoders so I had to replace the runtime exception with something,
+    	// but I'm not sure this is correct.
+    	return setAccess(0, forward, backward);
```

Comment:
Yes, `speedEncoder` is `null`. I'll leave `return setAccess(0, forward, backward);` for now, I've also filed #800 for this.